lib/config/rule-validator.js

```diff
@@ -169,7 +169,15 @@ class RuleValidator {
                 if (validateRule.errors) {
                     throw new Error(`Key "rules": Key "${ruleId}": ${
                         validateRule.errors.map(
-                            error => `\tValue ${JSON.stringify(error.data)} ${error.message}.\n`
+                            error => {
+                                if (error.keyword === "additionalProperties" && error.schema === false) {
+                                    const expectedProperties = Object.keys(error.parentSchema.properties).map(property => `"${property}"`);
+
+                                    return `\tValue ${JSON.stringify(error.data)} ${error.message}.\n\tUnexpected property "${error.params.additionalProperty}". Expected properties: ${expectedProperties.join(", ")}.\n`;
```

Comment:
```suggestion
                                    return `\tValue ${JSON.stringify(error.data)} ${error.message}.\n\t\tUnexpected property "${error.params.additionalProperty}". Expected properties: ${expectedProperties.join(", ")}.\n`;
```

Since the "unexpected property" line relates to the previous line, it might be good to add an indentation level to clarify that.

Reply:
`\t` is already there. Did you mean `\t\t`?